scripts/windows/test_task.bat

```diff
@@ -52,7 +52,7 @@ set PDOTEST_DSN=odbc:%ODBC_TEST_DSN%
 
 rem setup Firebird related exts
 curl -sLo Firebird.zip https://github.com/FirebirdSQL/firebird/releases/download/v3.0.9/Firebird-3.0.9.33560-0_x64.zip
-7z x -oC:\Firebird Firebird.zip
+7z x -y -oC:\Firebird Firebird.zip
```

Comment:
This would not be needed for a clean-room environment, as it should not be possible to overwrite files that did not exist.